core/src/service/index.ts

```diff
@@ -233,13 +236,27 @@ export function enhanceAPI<T extends FN>(_apiFn: T, config?: APIConfig<T>) {
       return body;
     });
 
+  /**
+   * when calling useData or useState, it may be called by multiple components at the same time,
+   * so when fetch finished, each setData function should be invoked one by one
+   * @param subscribeIdRef each subscribe maintains one unique id, used to unsubscribe when unmount component
+   * @param setData subscribe callback, which to set state data
+   */
+  const subscribe = (subscribeIdRef: React.MutableRefObject<string>, setData: Function) => {
+    if (!subscribeIdRef.current) {
+      const uid = `id_${++subscribeSeriesId}`;
```

Comment:
Prefer to use _.uniqueId('id_') as initial value.So not need subscribeSeriesId outside.